examples/web-ui/material/menus/menu.smash

```diff
@@ -0,0 +1,18 @@
+* Navigate to first level page {{page}}
+    setFirstLevelMenuItems +? {
+        props({
+            'hamburger': `body > header > button.header__hamburger.header__button`,
+            'layoutButton': `body > nav.side-nav.initialized > ul:nth-child(5) > li:nth-child(3) > button`
```

Comment:
Those are mighty long selectors :) Can you make them smaller, maybe using other props and `'text'`?

Reply:
...this is actually a problem I'm seeing in a lot of people's smash code. They're used to pasting in very long selectors or xpaths, even though it's faster and safer to use all the features available in elementfinders.